docs/docker-integration/cloudformation.rst

```diff
@@ -0,0 +1,167 @@
+			<span>Click "Next". Enter a <code>Stack name</code>. This can be any descriptive name.</span>
+			<span>Enter your <code>KeyName</code> from Step 1. Then enter your AWS <code>AccessKeyID</code> and <code>SecretAccessKey</code> credentials.</span><span>If you don't know these, <a href="javascript:void(0);" onclick="$('#iam-instructions').show();">click here</a>.</span>
+            <div id="iam-instructions" style="text-align:left; display:none;">
+                <span>You can generate new credentials on your <a href="https://console.aws.amazon.com/iam/home#users" target="_blank">IAM Users</a> page:</span>
+                <span><ul><li>Click on your user and go to the "Security Credentials" tab.</li><li>Click "Create Access Key".</li><li>Click "Show User Security Credentials".</li></ul></span>
+            </div>
+		</div>
+		<div class="step-stages__step">
+			<img src="../_images/12-cloudformation-settings.png" alt="Optionally fill in volumehub settings"/>
+            <span>Optionally, <a href="https://clusterhq.com/volumehub/" target="_blank">register for a Volume Hub account</a>. This provides a web interface to see what's going on in your cluster.</span>
+           <span>Once you're logged in to the Volume Hub, <a href="https://volumehub.clusterhq.com/v1/token" target="_blank">get your Volume Hub token here</a> and copy the <code>&lt;YourToken&gt;</code> part from <code>{"token": "&lt;YourToken&gt;"}</code> into the <code>VolumeHubToken</code> field.</span>
+		</div>
+		<div class="step-stages__step">
+			<img src="../_images/13-cloudformation-create.png" alt="Click create"/>
+			<span>Click "Next" twice and then click "Create" to create your cluster.</span>
+		</div>
+	</div>
+
+	<div class="step-stages step-stages--3up">
+		<div class="step-stages__excerpt">
+			<h2 class="step-stages__heading">The waiting bit!</h2>
+			<p>Wait for the cluster to come up. This can take 5-10 minutes.</p>
+		</div>
+		<div class="step-stages__step first">
+			<img src="../_images/21-refresh.png" alt="Refresh button on CloudFormation console"/>
+			<span>The stack may not show up immediately. Click the refresh button a few times to see it show up.</span>
+		</div>
+		<div class="step-stages__step">
+			<img src="../_images/22-create-in-progress.png" alt="Stack create in progress message"/>
+			<span>Once the stack shows up, it will stay in CREATE_IN_PROGRESS state for 5-10 minutes. Wait for it to transition to...</span>
+		</div>
+		<div class="step-stages__step">
+			<img src="../_images/23-create-complete.png" alt="Stack create create complete message"/>
+			<span>... CREATE_COMPLETE state.</span>
+		</div>
+	</div>
+
+	<div class="step-stages step-stages--3up">
+		<div class="step-stages__excerpt">
+			<h2 class="step-stages__heading">Step 3</h2>
+			<p>Verifying your installation.</p>
+            <p>Click on the "Outputs" tab for your stack. If this is not visible, click the drop down icon on the current tab.</p>
```

Comment:
I think words "the drop down icon" are unclear. 
We could probably remove that bit because its a standard AWS split screen UI feature.
Or refer to it using the AWS terminology: "minimize", "restore" and "maximize"